index.html

```diff
@@ -3,7 +3,11 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>AJAX Project</title>
+  <meta name="title" content="Agent Show Case">
+  <meta name="description" content="an AJAX application for Valorant players who want to understand each character's abilities and lore.">
+  <meta name="image" content="https://moamel-hashim.github.io/agent-show-case/images/logo.png">
+  <meta name="url" content="https://moamel-hashim.github.io/agent-show-case">
+  <title>Agent Show Case</title>
```

Comment:
These should use the `property` attribute not `name`, and the values need the `og` with them as outlined in the below image:
![Screen Shot 2022-05-26 at 3 02 00 PM](https://user-images.githubusercontent.com/11063357/170587277-e3d76396-98b5-44a2-b049-5510b52e5c75.png)